### DISCUSSION TITLE
Packages were not found in the pkg-config search path

### DISCUSSION THREAD

**Discussion by ganzobtn** [PyAV-Org/PyAV#995] [1. Help] [answered]
I upgraded python to 3.8 in my docker. When I try to install AV using pip3, it gives error

## Actual behavior

{{
 Complete output from command python setup.py egg_info:
    Package libavformat was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libavformat.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libavformat' found
    Package libavcodec was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libavcodec.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libavcodec' found
    Package libavdevice was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libavdevice.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libavdevice' found
    Package libavutil was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libavutil.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libavutil' found
    Package libavfilter was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libavfilter.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libavfilter' found
    Package libswscale was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libswscale.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libswscale' found
    Package libswresample was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libswresample.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libswresample' found
    pkg-config could not find libraries ['avformat', 'avcodec', 'avdevice', 'avutil', 'avfilter', 'swscale', 'swresample'] }}

**Comment by JensTimmerman** [comment, marked as answer]
install `libavformat-dev libavdevice-dev` or `libavformat-free-devel libavdevice-free-devel`
